### ISSUE TITLE
Verdi calculation list raises exception 

### ISSUE THREAD

**Issue by szoupanos** [aiidateam/aiida_core#476]
After checking out the release-0.8.0rc1, the verdi calculation list raises an exception under SQLA and Django

E.g.

SQLA
```
aiida@ubuntu-aiida-vm1:~/aiida-code/aiida_core$ verdi -p sqla3 calculation list
# Last daemon state_updater check: 23h:40m:39s ago (at 11:58:09 on 2017-03-22)
1 Raised when an error occurs during SQL compilation
2 (<class 'sqlalchemy.exc.CompileError'>, CompileError("Multiple, unrelated CTEs found with the same name: 'recentstates'",), <traceback object at 0x7f83403a3560>)
3 <class 'sqlalchemy.exc.CompileError'>
4 Multiple, unrelated CTEs found with the same name: 'recentstates'
5 <traceback object at 0x7f83403a3560> Sorry I mean line... 222
  File "/home/aiida/aiida-code/aiida_core/aiida/cmdline/commands/calculation.py", line 222, in calculation_list
    projections=parsed_args.project,
  File "/home/aiida/aiida-code/aiida_core/aiida/orm/implementation/general/calculation/job/__init__.py", line 982, in _list_calculations
    res = results_generator.next()
  File "/home/aiida/aiida-code/aiida_core/aiida/orm/querybuilder.py", line 2316, in iterdict
    for item in self._impl.iterdict(query, batch_size, self.tag_to_projected_entity_dict):
  File "/home/aiida/aiida-code/aiida_core/aiida/backends/sqlalchemy/querybuilder_sqla.py", line 730, in iterdict
    raise e
6 None
```

Django

```
aiida@ubuntu-aiida-vm1:~/aiida-code/aiida_core$ verdi -p dj2 calculation list
# Last daemon state_updater check: 1D:01h:29m ago (at 10:17:03 on 2017-03-22)
1 Inappropriate argument type.
2 (<type 'exceptions.TypeError'>, TypeError("column (<class 'sqlalchemy.sql.elements.Label'>) (SELECT db_dbcalcstate.state \nFROM db_dbcalcstate, (SELECT db_dbcalcstate.dbnode_id AS dbnode_id, max(db_dbcalcstate.time) AS lasttime \nFROM db_dbcalcstate, db_dbnode AS db_dbnode_1 \nWHERE db_dbcalcstate.dbnode_id = db_dbnode_1.id GROUP BY db_dbcalcstate.dbnode_id) AS anon_1, db_dbnode AS db_dbnode_1 \nWHERE db_dbcalcstate.time = anon_1.lasttime AND db_dbcalcstate.dbnode_id = db_dbnode_1.id) is not a valid column",), <traceback object at 0x7faf66dcb950>)
3 <type 'exceptions.TypeError'>
4 column (<class 'sqlalchemy.sql.elements.Label'>) (SELECT db_dbcalcstate.state
FROM db_dbcalcstate, (SELECT db_dbcalcstate.dbnode_id AS dbnode_id, max(db_dbcalcstate.time) AS lasttime
FROM db_dbcalcstate, db_dbnode AS db_dbnode_1
WHERE db_dbcalcstate.dbnode_id = db_dbnode_1.id GROUP BY db_dbcalcstate.dbnode_id) AS anon_1, db_dbnode AS db_dbnode_1
WHERE db_dbcalcstate.time = anon_1.lasttime AND db_dbcalcstate.dbnode_id = db_dbnode_1.id) is not a valid column
5 <traceback object at 0x7faf66dcb950> Sorry I mean line... 222
  File "/home/aiida/aiida-code/aiida_core/aiida/cmdline/commands/calculation.py", line 222, in calculation_list
    projections=parsed_args.project,
  File "/home/aiida/aiida-code/aiida_core/aiida/orm/implementation/general/calculation/job/__init__.py", line 982, in _list_calculations
    res = results_generator.next()
  File "/home/aiida/aiida-code/aiida_core/aiida/orm/querybuilder.py", line 2315, in iterdict
    query = self.get_query()
  File "/home/aiida/aiida-code/aiida_core/aiida/orm/querybuilder.py", line 2182, in get_query
    query = self._build()
  File "/home/aiida/aiida-code/aiida_core/aiida/orm/querybuilder.py", line 1977, in _build
    self._build_filters(alias, filter_specs)
  File "/home/aiida/aiida-code/aiida_core/aiida/orm/querybuilder.py", line 1214, in _build_filters
    in filter_operation_dict.items()
  File "/home/aiida/aiida-code/aiida_core/aiida/backends/djsite/querybuilder_django/querybuilder_django.py", line 277, in get_filter_expr
    expr = self.get_filter_expr_from_column(operator, value, column)
  File "/home/aiida/aiida-code/aiida_core/aiida/backends/djsite/querybuilder_django/querybuilder_django.py", line 108, in get_filter_expr_from_column
    type(column), column
6 None
```

**Comment by giovannipizzi** [comment]
Fixed in #479

**Comment by szoupanos** [comment]
Fix by #479 and tests by #481
